projects/libyuv/fuzz_common.cc

```diff
@@ -0,0 +1,68 @@
+/* # Copyright 2020 Google Inc. */
+/* # */
+/* # Licensed under the Apache License, Version 2.0 (the "License"); */
+/* # you may not use this file except in compliance with the License. */
+/* # You may obtain a copy of the License at */
+/* # */
+/* #      http://www.apache.org/licenses/LICENSE-2.0 */
+/* # */
+/* # Unless required by applicable law or agreed to in writing, software */
+/* # distributed under the License is distributed on an "AS IS" BASIS, */
+/* # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. */
+/* # See the License for the specific language governing permissions and */
+/* # limitations under the License. */
+/* # */
+/* ################################################################################ */
+
+#include <iostream>
+#include <fstream>
+
+#include "fuzz_common.h"
+#include <fuzzer/FuzzedDataProvider.h>
+#include "libyuv.h"
+
+using std::ofstream;
+using std::ios;
+
+bool write_conf(int width, int height, int cpu_flags)
+{
+// ios::trunc
+    ofstream out_file("libyuv_fuzz.conf", ios::out | ios::app);
+    if (out_file)
+    {
+        out_file << "width = " << width << " height= " << height << " cpu_flag = " << cpu_flags<<std::endl;
+        out_file.close();
+        return 1;
+    }
+    return 0;
+}
+
+int generate_cpuflags(int random_num)
+{
+    int cpu_info = 0;
+    if (random_num & 0x01)
+    {
+        //disable all cpu specific optimizations.
+        cpu_info = 1;
+        return cpu_info;
+    }
+
+    //enable all cpu specific optimizations.
+    cpu_info = -1;
+
+#if defined(__arm__) || defined(__aarch64__)
+    return cpu_info &= ~(random_num & 0x04);
```

Comment:
4 is not safe... it could change in future.
It would be safe to literally mask the flags with a random number, disabling parts of the cpuinfo.
e.g. disable avx.
Its safe to disable anything, just not enable anything.